src/main/java/net/fabricmc/loader/launch/FabricTweaker.java

```diff
@@ -88,13 +101,19 @@ public void injectIntoClassLoader(LaunchClassLoader launchClassLoader) {
 
 		launchClassLoader.addClassLoaderExclusion("net.fabricmc.api.Environment");
 		launchClassLoader.addClassLoaderExclusion("net.fabricmc.api.EnvType");
+		launchClassLoader.addClassLoaderExclusion("net.fabricmc.api.ModInitializer");
+		launchClassLoader.addClassLoaderExclusion("net.fabricmc.api.ClientModInitializer");
+		launchClassLoader.addClassLoaderExclusion("net.fabricmc.api.DedicatedServerModInitializer");
```

Comment:
ClientModInitializer and DedicatedServerModInitializers have `@Environment` annotations. Is this exclusion going to affect their class transformation?

Reply:
Nevermind, they don't.